pkg/ui/src/views/reports/containers/stores/index.tsx

```diff
@@ -122,8 +120,8 @@ class Stores extends React.Component<StoresProps, {}> {
         <h1>Stores</h1>
         <h2>{header} stores</h2>
         {
-          _.map(stores.stores, (store, key) => (
-            this.renderStore(store, key)
+          _.map(_.sortBy(stores.stores, (store) => store.store_id), (store) => (
```

Comment:
better to move the sort into the selector so we're not redoing it needlessly

Reply:
I don't actually know what this means.